tests/jest/unitTest/UI_layer/Header.test.tsx

```diff
@@ -0,0 +1,32 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import { Header } from "../../../../src/Components/Navigation/Header";
```

Comment:
```suggestion
import { Header } from "@components/Navigation/Header";
```
could use https://github.com/components here